deployments/helm/nvidia-dra-driver-gpu/Chart.yaml

```diff
@@ -15,10 +15,10 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: v25.3.0
+version: 25.3.0-rc.1
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "v25.3.0"
+appVersion: "25.3.0-rc.1"
```

Comment:
@klueska

Reply:
Do you have a specific question @ArangoGutierrez?

Note that dropping the `v` is consistent with the device plugin: https://github.com/NVIDIA/k8s-device-plugin/blob/f171c9269279c5021b286c9a7ca9a428bb673bea/deployments/helm/nvidia-device-plugin/Chart.yaml#L5-L6

Reply:
I was pointing Kevin after his question https://github.com/NVIDIA/k8s-dra-driver-gpu/pull/277#issuecomment-2714862744

Reply:
I see. I was reviewing commit-by-commit and expected the `v` to be stripped as part of the first commit for `[Makefile: make VERSION not have v prefix](https://github.com/NVIDIA/k8s-dra-driver-gpu/pull/277/commits/c1cd7c69c64a3c44f6a073ee1613a4d77a14460b)`

Reply:
>  dropping the v is consistent with the device plugin

Thanks!

> Do you have a specific question @ArangoGutierrez?

For the record, this ping was in response to https://github.com/NVIDIA/k8s-dra-driver-gpu/pull/277#issuecomment-2714862744.